### PR TITLE
Feature/header container

### PR DESCRIPTION
**Pull request by DimaPolishuk1995**

### CODE REVIEW

File: src/components/containers/NavSearch/NavSearch.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const NavSearch = props => <div className="wrap-search wrap-header-search">
+  <div className="header-search">
+    <div className="control-group left control-search-select dropdown">
+      <button className="btn btn-dropdown-toggle btn-search-toggle"><i className="btn-icon fa fa-bars"/></button>
+      <div className="dropdown-menu dropdown-menu-search-select dropdown-menu-panel dropdown-menu-left dropdown-menu-small-size">
+        <div className="heading">Search Options</div>
+        <div className="dropdown-menu-wrap-list">
```
Comment (polesskiy-dev) [inline src/components/containers/NavSearch/NavSearch.js]: it should be separate component - <PRDropDown/> for example

File: src/components/containers/TopHeader/TopHeader.js [hunk 2]
```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import MainLogo from '../MainLogo/MainLogo';
+import NavSearch from '../NavSearch/NavSearch';
+import UserAccountPanel from '../UserAccountPanel/UserAccountPanel';
 
 const TopHeader = props => <div className="navbar">
-
+  <MainLogo />
+  <UserAccountPanel />
+  <NavSearch />
```
Comment (polesskiy-dev) [inline src/components/containers/TopHeader/TopHeader.js]: Good.

File: src/components/containers/UserAccountPanel/UserAccountPanel.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PTButtonIcon from '../PTButtonIcon/PTButtonIcon';
+
+const UserAccountPanel = props => <ul className="user-panel" role="tablist">
```
Comment (polesskiy-dev) [inline src/components/containers/UserAccountPanel/UserAccountPanel.js]: Doesn't looks like a panel. Can you provide a screenshots?

File: src/components/containers/PTButtonIcon/PTButtonIcon.js [hunk 4]
```diff
@@ -1,18 +1,5 @@
...
-    }
-    return (
-      <a className={`btn-header ${ this.props.classButtonName }`}>
-        <i className={`fa ${ this.props.icon }`}/>
-        {span}
-      </a>
-    );
-  }
-}
+const PTButtonIcon = ({children, className, ...restProps}) => <button className={className} {...restProps}>{children}</button>;
```
Comment (polesskiy-dev) [inline src/components/containers/PTButtonIcon/PTButtonIcon.js]: Looks like now it's just PTButton

File: src/components/containers/PTButtonIcon/PTButtonIcon.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const PTButtonIcon = ({children, className, ...restProps}) => <button className={className} {...restProps}>{children}</button>;
```
Comment (polesskiy-dev) [inline src/components/containers/PTButtonIcon/PTButtonIcon.js]: Rename to PTButton - we don't use any icon reference here

File: src/components/containers/PTButtonIcon/PTButtonIcon.js (hunk 4 above)
Comment (DimaPolishuk1995) [inline src/components/containers/PTButtonIcon/PTButtonIcon.js]: ok

File: src/components/containers/PTButtonIcon/PTButtonIcon.js (hunk 5 above)
Comment (DimaPolishuk1995) [inline src/components/containers/PTButtonIcon/PTButtonIcon.js]: ok

**Pull request merged**